src/api.ts

```diff
@@ -0,0 +1,16 @@
+const API_URL = 'https://jsonplaceholder.typicode.com';
+
+const getItems = async<T>(url: string): Promise<T[]> => {
+  const response = await fetch(API_URL + url);
+
+  if (!response.ok) {
```

Comment:
For future debugging, we can add here some warning that will show a message in console.
Check this please:
 https://www.w3schools.com/jsref/met_console_warn.asp 
 or https://www.w3schools.com/jsref/met_console_error.asp
 I think it could look like console.error('API, getItems - Error fetching data:', response.status);